.travis/custom_release.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+NODE_ENV=production npm run build
```

Comment:
`NODE_ENV=production` is not required. The npm script is setting this variable.

Reply:
This was following what we had in the release scripts for catalog/approval.  Here is the initial catalog script: https://github.com/RedHatInsights/approval-ui/blob/6e1ec589cdd110c5b2c7e5ce490aadd8e25f401d/.travis/custom_release.sh

If you are saying it can be removed that's great.  Sounds like we should remove from all the UI release scripts, correct?

Reply:
Correct. The env variable is set by the npm script in all our UI repositories.
```
"build": "NODE_ENV=production webpack --config config/webpack.config.js --env.prod=true --mode production",
```